src/compiler.js

```diff
@@ -70,7 +70,7 @@ class Compiler {
     try {
       return Subgraph.load(this.options.subgraphManifest)
     } catch (e) {
-      throw Error('Failed to load subgraph')
+      throw Error(`Failed to load subgraph: ${e.message}`)
```

Comment:
Thank you so much for this!! I was adding it myself any time I got that error message

Reply:
Yeah, I know 👍